src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,16 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[] {
+                new Bulldozer("Buldozer"),
+                new Excavator("Excavator"),
+                new Truck("Truck")
+        };
+
+        for (int i = 0; i < machines.length; i++) {
+            machines[i].doWork();
+            machines[i].stopWork();
```

Comment:
While the code inside the loop is correct, consider enhancing the readability of your loop by using the enhanced for-loop (for-each loop). This will make your code cleaner and easier to understand.